extensions/interactions/ItemSelectionInput/directives/item_selection_input_response_directive.html

```diff
@@ -2,12 +2,19 @@
   <em>No choices selected.</em>
 </div>
 
-<div ng-if="answer.length === 1">
+<div ng-if="answer.length === 1" class="item-selection-input-response-content">
   <div angular-html-bind="answer[0]"></div>
 </div>
 
-<ul ng-if="answer.length > 1">
+<ul ng-if="answer.length > 1" class="item-selection-input-response-content">
   <li ng-repeat="response in answer track by $index">
     <div angular-html-bind="response"></div>
   </li>
 </ul>
+
+
+<style>
+  .item-selection-input-response-content img {
+    max-height: 150px;
```

Comment:
I think maybe this could be a bit smaller?

Reply:
150->100
![150_2_100](https://user-images.githubusercontent.com/8732147/34091888-5934067a-e375-11e7-83cb-a92f7ee3cfc1.png)